app/build.gradle

```diff
@@ -43,18 +46,33 @@ android {
         main {
             kotlin {
                 srcDirs += 'build/generated/source/navigation-args'
+                java.srcDirs += 'build/generated/data_binding_base_class_source_out'
             }
         }
     }
 }
 
+tasks.getByPath("preBuild").dependsOn("ktlintFormat")
+
+ktlint {
+    android = true
+    ignoreFailures = false
+    disabledRules = ["max-line-length"]
+    reporters {
+        reporter "plain"
+        reporter "checkstyle"
+        reporter "sarif"
+    }
+}
+
```

Comment:
What is it?

Reply:
https://github.com/pinterest/ktlint
https://youtu.be/hSgPNbEcX98